compiler/luci/pass/src/FuseInstanceNormPass.cpp

```diff
@@ -114,8 +115,6 @@ bool NodeFiller<ARG_TYPE_1, ARG_TYPE_2>::with_commutative_args_of(const COMM_NOD
 } // namespace
 
 // Helper to check detail
-namespace
```

Comment:
Q) is there any particular reason to remove this?

Reply:
Yes, I removed this so I can write unit tests for these functions in *.test.cpp file